### PR TITLE
Add aws release 17.0.0-alpha1

### PR DESCRIPTION
**Pull request by whites11**

### CODE REVIEW

**Review by alex-dabija — APPROVED** [review]

**Comment by calvix** [comment]
I would still wait for the cgroupv1 for this release as well

**Review by fiunchinho — APPROVED** [review]

File: aws/v17.0.0-alpha1/release.yaml [hunk 1]
```diff
@@ -50,15 +50,15 @@ spec:
     version: 1.10.1-nftables
   - name: aws-operator
     releaseOperatorDeploy: true
-    version: 10.13.0
+    version: 10.15.0
```
Comment (whites11) [inline aws/v17.0.0-alpha1/release.yaml:53]: This requires testing from scratch the release

**Comment by calvix** [comment]
docs are out
https://docs.giantswarm.io/advanced/forcing-cgroupsv1/

handing over @alex-dabija

**Pull request merged**